.gitignore

```diff
@@ -1,5 +1,6 @@
 .DS_Store
 
 credentials/*.pem
+credentials/private-key.json
```

Comment:
Rename this to use a more approach signifier. If this is a gcloud key, it should be clear in the naming.

We should also consider making all credentials/* ignored. If so, make sure there is a .gitkeep in the credentials folder so it is maintained in git.